message/html/render/post.js

```diff
@@ -18,7 +18,7 @@ exports.create = function (api) {
     if (msg.value.content.type !== 'post') return
     var element = api.message.html.layout(msg, extend({
       title: messageTitle(msg),
-      content: messageContent(msg),
+      content: msg.isBlocked ? 'Content of a blocked user' : messageContent(msg),
       layout: 'default'
```

Comment:
here we could actually pivot into a different layout if we wanted.
Would make it nice and easy for different clients to choose how they wanted to handle it